### PR TITLE
feat:added units to capacitance measurement

### LINKED ISSUE #1164: Add units to capacitance measurement
**Actual Behaviour**

Currently the capacitors values are shown in the form of power of e (scientific form) 

**Expected Behaviour**

The capacitors should be shown with units such as pico farad or nano farad etc

**Steps to reproduce it**

Go to MultimeterAcitivity and change it.

**LogCat for the issue**

Not applicable

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by abhinavraj23**
Fixes #1164 

**Changes**:
Added units namely pico, nano, micro and milli farad to the measurement and also reduced the decimal places to two.

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[units.apk.zip](https://github.com/fossasia/pslab-android/files/2152584/units.apk.zip)

### CODE REVIEW

**Comment by abhinavraj23** [comment]
@CloudyPadmal @harsh-2711 @Vikum94 @Avjeet Please review

File: app/src/main/java/org/fossasia/pslab/activity/MultimeterActivity.java [hunk 1]
```diff
@@ -149,7 +149,26 @@ public void onClick(View view) {
...
+                    String capacitanceUnit;
+                    if (capacitance < 1e-9) {
+                        Capacitance = capacitanceFormat.format((capacitance / 1e-12));
+                        capacitanceUnit = "picoFarad";
+                    } else if (capacitance < 1e-6) {
+                        Capacitance = capacitanceFormat.format((capacitance / 1e-9));
+                        capacitanceUnit = "nFarad";
+                    } else if (capacitance < 1e-3) {
+                        Capacitance = capacitanceFormat.format((capacitance / 1e-6));
+                        capacitanceUnit = "microFarad";
```
Comment (CloudyPadmal) [inline app/src/main/java/org/fossasia/pslab/activity/MultimeterActivity.java]: These are not units. Refer https://en.wikipedia.org/wiki/Orders_of_magnitude_(capacitance)

**Comment by CloudyPadmal** [comment]
@abhinavraj23 can you open a similar issue to resistance and implement SI units there as well?

**Comment by abhinavraj23** [comment]
@CloudyPadmal But it is already done in case of resistance measurement

**Comment by CloudyPadmal** [comment]
@abhinavraj23 I mean the units format. They are in "Ohms" not the symbol.

**Pull request merged**